src/functions/attestor/attestor-request.functions.ts

```diff
@@ -0,0 +1,83 @@
+import { isEmpty, join } from 'ramda';
+import {
+  FundingTXAttestorInfo,
+  WithdrawDepositTXAttestorInfo,
+} from 'src/models/attestor.models.js';
+import { AttestorError } from 'src/models/errors.js';
+
+export async function sendRequest(url: string, body: string): Promise<boolean | string> {
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': '*' },
+    body,
+  });
+  if (!response.ok) {
+    throw new AttestorError(`Attestor Response ${url} was not OK: ${response.statusText}`);
+  }
+  return true;
+}
+
+export async function submitFundingPSBT(
```

Comment:
Not a priority, but it seems you could DRY up these two functions, since only the URLS and the TxInfo DTOs are different, all other logic is shared